core/src/main/java/com/novoda/landingstrip/LandingStrip.java

```diff
@@ -185,8 +200,11 @@ private void forceDrawIndicatorAtPosition(int position) {
         invalidate();
     }
 
+    public void detach(ViewPager viewPager) {
```

Comment:
This is the method https://github.com/novoda/landing-strip/pull/23/files#diff-03f70e68af4374fc21ddd30d94ca1309L72

```
public void attach(ViewPager viewPager) {
         attach(viewPager, viewPager.getAdapter());
     }
```